### DISCUSSION TITLE
velero backuplocation for CSI plugin

### DISCUSSION THREAD

**Discussion by vijaynetha** [vmware-tanzu/velero#4292] [Community support Q&A]
Hi Team,

i have deployed velero to backup and restore using CSI plugin.
I am using below command.

**velero install --provider csi --plugins registry.app.corpintra.net/velero/velero-plugin-for-csi:v0.2.0 --bucket <bucketname> --secret-file credentials-velero --use-volume-snapshots=true --snapshot-location-config region=emea --backup-location-config region=emea,s3ForcePathStyle="true",s3Url=https://url/**

I velero pod logs i can error related to Backupstore location. Please help me on this issue.

time="2021-10-28T06:34:30Z" level=error msg="Error getting a backup store" backup-storage-location=default controller=backup-storage-location error="unable to locate ObjectStore plugin named velero.io/csi" **logSource="pkg/controller/backup_storage_location_controller.go:100**

**Comment by chrislinan** [comment]
Hi @vijaynetha 
CSI plugin can't work alone, it's a general plugin works with other cloud provider plugin. So, you can not set the `provider` as csi. for more detail you can refer to this article : https://gist.github.com/ashish-amarnath/b4162d2836a63516540bfa1d24783120
